sys/src/9/port/devramfs.c

```diff
@@ -208,19 +273,44 @@ ramread(Chan *c, void *buf, int32_t n, int64_t off)
 	return n;
 }
 
+typedef double Align;
+typedef union Header Header;
+
+union Header {
+        struct {
+                Header* next;
+                uint    size;
+        } s;
+        Align   al;
```

Comment:
what's this for again? To force the allocation alignment? There's better ways to do this, but most allocators currently (or should) align to 64 bytes anyway?